Utilities/bootstrap

```diff
@@ -547,27 +547,40 @@ def get_swiftpm_flags(args):
             "-Xlinker", "-rpath",
             "-Xlinker", "@executable_path/../../../../../SharedFrameworks",
 
-            # For TensorFlow.
```

Comment:
Note: this reverts https://github.com/apple/swift-package-manager/pull/2548, which didn't fix https://github.com/tensorflow/swift/issues/347.

This change didn't work because the code is nested under the `if args.llbuild_link_framework:` condition, which doesn't quite make sense.